frontend/src/components/Pages/MissionPage/MapPosition/MissionMapView.tsx

```diff
@@ -4,29 +4,34 @@ import { Mission } from 'models/Mission'
 import { useCallback, useEffect, useRef, useState } from 'react'
 import styled from 'styled-components'
 import NoMap from 'mediaAssets/NoMap.png'
-import { defaultPose, Pose } from 'models/Pose'
 import { PlaceRobotInMap, PlaceTagsInMap } from '../../../../utils/MapMarkers'
 import { BackendAPICaller } from 'api/ApiCaller'
 import { TaskStatus } from 'models/Task'
+import { MapCompass } from 'utils/MapCompass'
 
 interface MissionProps {
     mission: Mission
 }
 
 const MapCard = styled(Card)`
     display: flex;
-    height: 600px;
     width: 600px;
```

Comment:
```suggestion
    max-width: 50%;
```
Or some other percentage, but this makes the map scale with window size

Reply:
Hei, the max-width in this case is relative to the size of the card which is 600x600px so what happens is that the map can have a maximum height of 100% which is 600px. This is important because for the max width I want it to be only 90% so we have space to include the compass and to make sure the size of the map is never bigger than the card! I also like the percentages because it gets more obvious that is in reference to the card size and if you ever change the card size it will adapt and you don't have to change as many things